bpf/tests/bpf_nat_tests.c

```diff
@@ -1302,4 +1307,107 @@ int test_nat4_port_allocation_udp_check(struct __ctx_buff *ctx)
 	test_finish();
 }
 
-BPF_LICENSE("Dual BSD/GPL");
```

Comment:
Why is the license declaration being removed?

Reply:
Removed by mistake.